sql/src/test/java/io/crate/testing/SQLPrinter.java

```diff
@@ -49,6 +50,12 @@ public static String print(Object o) {
             return print((Collection<Symbol>) o);
         } else if (o == null) {
             return "null";
+        } else if (o instanceof QueryClause) {
+            if (((QueryClause) o).hasQuery()) {
```

Comment:
When we use isSQL() with that we cannot distinguish between MATCH_ALL and NO_MATCH.
I guess we would need something like:
```
QueryClause qc = (QueryClause) o;
if (qc.hasQuery()) {
    return print(((QueryClause) o).query());
} else if (qc.noMatch()) {
    return "NO_MATCH";
} else {
    return "MATCH_ALL";
}
```
or would it break something else?

Reply:
@matriv pushed a fixup for this

Reply:
Thanks! acked!